.github/workflows/default.yaml

```diff
@@ -44,9 +44,8 @@ jobs:
 
       - run: ./bootstrap.sh
       - run: ./configure --with-openssl
-      - run: make -j2
```

Comment:
Is there a better way to trigger parallel builds than remembering to specify `-j...` every time we ran "make"? Perhaps we should set MAKEFLAGS or use some kind of autoconf/automake tricks?

Reply:
I share Francesco's concern. FWIW, IIRC, in some environments I have tested (with _more_ than n=2 CPU cores), `-j` was noticeably slower than `-jn`.

I am not going to object to bare `-j` if others prefer that but please make this decision based on GitHub Actions CI tests.